sbin/releaseCheck.sh

```diff
@@ -15,7 +15,7 @@ curl -q https://github.com/adoptium/temurin${TEMURIN_VERSION}-binaries/releases/
 #### LINUX (ALL)
 for ARCH in x64 aarch64 ppc64le s390x arm; do
   # jre, jdk, debugimage, static-libs (Except JDK8) in base, json, sha256
-  EXPECTED=12; [ "${TEMURIN_VERSION}" -eq 8 ] && EXPECTED=9
+  EXPECTED=16; [ "${TEMURIN_VERSION}" -eq 8 ] && EXPECTED=12
```

Comment:
Worth commenting what these magic numbers are?

Reply:
That's what the previous line is for - each of the four deliverables multiplied by the number of files associated with each one (although I agree I missed adjusting that to include the signature file!)

Reply:
Memo to self - the values may be incorrect now sine they won't include the SBOM files. Once this goes in I can verify that and raise a subsequent PR.

This tool is mostly just a convenience for the release champion to check what is (really) outstanding as a check against the status tables which say what /should/ be complete ;-)